include/openmc/geometry.h

```diff
@@ -35,6 +35,12 @@ struct BoundaryInfo {
   std::array<int, 3> lattice_translation {}; //!< which way lattice indices will change
 };
 
+//==============================================================================
+//! Check two distances by coincidence tolerance
+//==============================================================================
+
+bool coincident(const double& d1, const double& d2);
```

Comment:
Short types should be passed by value:
```C++
bool coincident(double d1, double d2);
```

Reply:
I think we should probably also put the definition of `coincident` here in the .h file so the compiler can inline it (since the definition is so short).